internal/gen-atomicwrapper/main.go

```diff
@@ -252,7 +249,7 @@ func (x *{{ .Name }}) Load() {{ .Type }} {
 	{{ if .Unpack -}}
 		return {{ .Unpack }}(x.v.Load())
 	{{- else -}}
-		var o {{ .Type }}{{ with .Zero }} = {{ . }}{{ end}}
+		var o {{ .Type }}
```

Comment:
don't think we need this variable anymore, since we have the zero value
```
if v := [..]; v != nil {
  return v.(type)
}
return _zero{{ .Type }}
```

Reply:
Ah, yeah good catch.